packages/core/src/LottieView/index.web.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { forwardRef, useImperativeHandle, useRef, useCallback } from 'react';
+import type { LottieViewProps } from '../types';
+import { DotLottiePlayer } from '@dotlottie/react-player';
+import { Player } from '@lottiefiles/react-lottie-player';
+import { parsePossibleSources } from './utils';
+
+const LottieView = forwardRef(
+  (
+    {
+      source,
+      speed,
+      loop,
+      webStyle,
+      autoPlay,
+      hover,
+      direction,
+      progress,
+      onAnimationFailure,
+      onAnimationFinish,
+      onAnimationLoop,
+    }: LottieViewProps,
+    ref: any,
+  ) => {
+    const sources = parsePossibleSources(source);
+    const isLottie =
+      sources.sourceName?.includes('.lottie') || !!sources.sourceDotLottieURI;
+    const lottieSource = sources.sourceDotLottieURI || sources.sourceName;
+
+    if (progress && __DEV__) {
```

Comment:
What if `progress === 0`?
```suggestion
    if (progress !== undefined && __DEV__) {
```